### PR TITLE
Fix multiselect on Mac: support for metaKey (command)

### PR DESCRIPTION
**Pull request by andywerner**
Multiselect was not working on Mac since ctrl-key + click triggers context menu to open.
Mac user want to use command + click anyway. 
Now it's working. Also added command key support in other places as well.

### CODE REVIEW

**Comment by andywerner** [comment]
@khaledsherkawi Please double check and test to make sure I did not break the PC side...
Then merge if it seems okay to you. Do not wait on teja's approval.
@hungerpirat Please have a short look on these changes even after we merged it. Just to be sure.

BTW: tested n my Mac on Chrome, Firefox and Safari

**Review by khaledsherkawi — APPROVED** [review]

**Pull request merged**
